content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -331,17 +331,34 @@ To enable the plugin, configure the following flags on the API server:
 
 ##### Using Authentication Configuration
 
-{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.30" state="beta" >}}
 
 JWT Authenticator is an authenticator to authenticate Kubernetes users using JWT compliant tokens. The authenticator will attempt to
 parse a raw ID token, verify it's been signed by the configured issuer. The public key to verify the signature is discovered from the issuer's public endpoint using OIDC discovery.
 
-The API server can be configured to use a JWT authenticator via the `--authentication-config` flag. This flag takes a path to a file containing the `AuthenticationConfiguration`. An example configuration is provided below.
-To use this config, the `StructuredAuthenticationConfiguration` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-has to be enabled.
+The minimum valid JWT payload must contain the following claims:
+```yaml
+{
+  "iss": "https://example.com",   // must match the issuer.url
+  "aud": ["my-app"],              // at least one of the entries in issuer.audiences must match the "aud" claim in presented JWTs.
+  "exp": 1234567890,              // token expiration time
+  "<username-claim>": "user"      // this is the username claim configured in the claimMappings.username.claim or claimMappings.username.expression
+}
+```
+
+The configuration file approach allows you to configure multiple JWT authenticators, each with a unique `issuer.url` and `issuer.discoveryURL`. The configuration file even allows you to specify [CEL](/docs/reference/using-api/cel/)
+expressions to map claims to user attributes, and to validate claims and user information. The API server also automatically reloads the authenticators when the configuration file is modified. An example configuration is provided below.
+
+You must specify the path to the authentication configuration using the `--authentication-config` flag on the API server. If you want to keep using command line flags instead of the configuration file, those will continue to work as-is.
+To gain access to new capabilities like configuring multiple authenticators, setting multiple audiences for an issuer, switch to using the configuration file.
+
+Starting Kubernetes 1.30, the configuration file format is beta-level, and only requires specifying `--authentication-config` since the `StructuredAuthenticationConfiguration` feature gate is enabled by default.
 
 {{< note >}}
-When the feature is enabled, setting both `--authentication-config` and any of the `--oidc-*` flags will result in an error. If you want to use the feature, you have to remove the `--oidc-*` flags and use the configuration file instead.
+If you want to keep using command line flags to configure JWT authenticator instead of a configuration file, those will continue to work as-is.
+
```

Comment:
Avoid “keep”; this carries an implication that there is an existing cluster being upgraded. In fact, people might deploy a brand new cluster.